app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Same here and as the last reviewer mentioned, please edit this line and create posts migration to make the foreign key `author_id` instead of `user_id`